frontend/src/components/Cases/CaseItem.tsx

```diff
@@ -97,10 +100,46 @@ const CaseContent = ({
   );
 };
 
-const CaseItem = ({ seq, caseData, implementations }: CaseProps) => {
+const CaseItem = ({
+  seq,
+  caseData,
+  implementations,
+  searchText,
+}: CaseProps) => {
   const [content, setContent] = useState(<></>);
   const [, startTransition] = useTransition();
   const schemaDisplayRef = useRef<HTMLDivElement>(null);
+  const { isDarkMode } = useContext(ThemeContext);
+
+  const highlightDescription = (
+    description: string,
+    searchText: string,
+  ): JSX.Element => {
+    if (!searchText) {
+      return <>{description}</>;
+    }
+
+    const regex = new RegExp(`(${searchText})`, "gi");
+    const parts: string[] = description.split(regex);
+    return (
```

Comment:
Yes! I have seen the problem. But I am not sure how to fix it. Can you suggest anything?

Reply:
Unfortunately had no ideas for fixing this. But I'd suggest looking up some sites/libs with "highlighting" functionality and seeing in browser how it looks like in DOM - maybe they are using different tags or some CSS rules to make it work?

Reply:
Did you manage to fix it?

Reply:
I have researched on it, and found https://www.npmjs.com/package/react-highlight-words, seems like it can potentially solve the problem. Can we do that?